source/index.ts

```diff
@@ -97,10 +99,19 @@ client.on("message", async (message) => {
 
 client.on("messageReactionAdd", async (reaction, user) => {
 
-	if (reaction.message.channel.type !== "text" || user.bot) return;
-
-	pluginManager.runEmojiCommand(reaction, user as Discord.User);
-
+	if (!(reaction.message.channel.type === ChannelType.GuildText || reaction.message.channel.type === ChannelType.GuildPublicThread ||  reaction.message.channel.type === ChannelType.GuildPrivateThread || reaction.message.channel.type === ChannelType.DM) || user.bot) return;
+	if(reaction.partial){
```

Comment:
```suggestion
	if (reaction.partial) {
```

please add spaces between brackets and statements. Only exception for this are function calls